docker-compose-mariadb.yml

```diff
@@ -1,5 +1,5 @@
 mariadb:
-  image: eventuateio/eventuate-mariadb:0.9.0.RC3
+  image: test-eventuate-mariadb
```

Comment:
I think it would make sense for builds on 'master' to publish BUILD-SNAPSHOT images (as well as libraries)

Reply:
Should I implement it?

Reply:
Yes. https://github.com/eventuate-foundation/eventuate-common/issues/33

Reply:
Also, please confirm that the code changes work with the current image versions, e.g. eventuateio/eventuate-mariadb:0.9.0.RC3